fileonfeature.txt

```diff
@@ -1,2 +1,3 @@
 this is my file on feature branch
-added soem lines to addited file
\ No newline at end of file
+added soem lines to addited file
+new line from new branch
```

Comment:
this is not the way how we add new lines to our txt file!
it should start with "added" for e.g.